src/caustics/lenses/epl.py

```diff
@@ -106,14 +109,17 @@ def __init__(
         phi: Optional[Tensor]
             Position angle of the lens.
             If not provided, it is considered as a free parameter.
+        *Unit: radians*
```

Comment:
The spacing doesn't match the rest of the usages.
There are missing units above.